### PR TITLE
feat(numericRefinementList)

### PR DESCRIPTION
**Pull request by maxiloc**
Created a new PR, did some mess with the first one.

- I updated the documentation (it will need a version of instant search which include this commit to actually work)
- I changed link by label + radio button for the default template

### CODE REVIEW

File: components/RefinementList/RefinementList.js [hunk 1]
```diff
@@ -35,7 +35,14 @@ class RefinementList extends React.Component {
       [this.props.cssClasses.active]: facetValue.isRefined
     });
 
-    let key = facetValue[this.props.facetNameKey] + '/' + facetValue.isRefined + '/' + facetValue.count;
+    let key = facetValue[this.props.facetNameKey];
+    if (facetValue.count !== undefined) {
```
Comment (redox) [inline components/RefinementList/RefinementList.js]: Bad copy/paste, you want to test `facetValue. isRefined` here.

File: docs/documentation.md [hunk 2]
```diff
@@ -532,6 +532,54 @@ results that have either the value `a` or `b` will match.
...
+instantsearch.widgets.numericRefinementList(options);
+{% endhighlight %}
+
+{% include widget-jsdoc/refinementList.md %}
+  </div>
+</div>
+
+<img class="widget-icon pull-left" src="../img/icon-widget-refinement.svg">
+This filtering widget lets the user choose one or multiple values for a single faceted attribute. You can specify if you want to filters to be ORed or ANDed. For example, if you filter on `a` and `b` with `OR`, the
+results that have either the value `a` or `b` will match.
```
Comment (redox) [inline docs/documentation.md]: we need to change the text of the doc here, it doesn't fit the numericRefinementList, copy/pasted I assume.

**Comment by redox** [comment]
I did 2 comments, but overall very nice @maxiloc :+1:

**Comment by vvo** [comment]
Wait what is docs/js/instantsearch.js?

**Comment by redox** [comment]
> Wait what is docs/js/instantsearch.js? It looks like a dist/ you commited.

Ah right, that was a temporary; need to be removed.

**Comment by maxiloc** [comment]
I rebased with the 3 fix

File: npm-shrinkwrap.json [hunk 3]
```diff
@@ -12453,4 +12453,4 @@
       }
     }
   }
-}
+}
```
Comment (vvo) [inline npm-shrinkwrap.json]: :trollface:

**Comment by vvo** [comment]
Looks..Like.. LGTM! :shipit: :rocket:

File: npm-shrinkwrap.json (hunk 3 above)
Comment (maxiloc) [inline npm-shrinkwrap.json]: Weird I did not touch this file,
Should I add the newline again ?

Comment (vvo) [inline npm-shrinkwrap.json]: No we are good, the next shrinkwrap will just do it

**Comment by vvo** [comment]
@redox I am good here, since you added comments I'll let you merge, again awesome feature :D

**Comment by redox** [comment]
:+1:

**Pull request merged**